noir-projects/noir-contracts/contracts/app/simple_token_contract/src/main.nr

```diff
@@ -153,11 +153,11 @@ pub contract SimpleToken {
             to,
         ));
 
-        emit_event_in_private_log(
+        emit_event_in_private(
             Transfer { from, to, amount },
             &mut context,
             to,
-            PrivateLogContent.NO_CONSTRAINTS,
+            MessageDelivery.UNCONSTRAINED_ONCHAIN,
```

Comment:
This does read much better, yeah. We may want to iterate on the names of the variants.